app/views/wizards/claims/add_user_wizard/_check_your_answers_step.html.erb

```diff
@@ -0,0 +1,36 @@
+<% content_for :page_title, sanitize(t(".page_title", caption: contextual_text)) %>
+
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= form_for(current_step, url: current_step_path, method: :put) do |f| %>
+        <%= f.govuk_error_summary %>
+
+        <span class="govuk-caption-l"><%= contextual_text %></span>
+        <h1 class="govuk-heading-l"><%= t(".title") %></h1>
+
+        <%= govuk_summary_list(actions: false) do |summary_list| %>
+          <% summary_list.with_row do |row| %>
+            <% row.with_key(text: User.human_attribute_name("first_name")) %>
+            <% row.with_value(text: wizard.steps[:user].first_name) %>
+          <% end %>
+          <% summary_list.with_row do |row| %>
+            <% row.with_key(text: User.human_attribute_name("last_name")) %>
+            <% row.with_value(text: wizard.steps[:user].last_name) %>
+          <% end %>
+          <% summary_list.with_row do |row| %>
+            <% row.with_key(text: User.human_attribute_name("email")) %>
+            <% row.with_value(text: wizard.steps[:user].email) %>
+          <% end %>
+        <% end %>
+
+        <div class="govuk-warning-text">
+          <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+          <strong class="govuk-warning-text__text"><%= t(".warning", user_name: "#{wizard.steps[:user].first_name} #{wizard.steps[:user].last_name}", school_name: @school.name) %></strong>
```

Comment:
No biggie, but if you wanted to, you could actually make use of [`User#full_name`](https://github.com/DFE-Digital/itt-mentor-services/blob/c46c9ffe69761c9d8f4389e43e41bd0659f1ebe1/app/models/user.rb#L38-L40) here.

```suggestion
          <strong class="govuk-warning-text__text"><%= t(".warning", user_name: wizard.steps[:user].user.full_name, school_name: @school.name) %></strong>
```